index.html

```diff
@@ -84,14 +84,159 @@ <h1>Hey there.<span class="span__br"><br /></span> I’m Mateo.
             </ul>
           </div>
 
-      <div class="arrow_button">
-        <a href="#">
-          <img src="./assets/button/arrowbutton.svg">
-        </a>
+        <div class="arrow_button">
+          <a href="#">
+            <img src="./assets/button/arrowbutton.svg">
+          </a>
+        </div>
       </div>
     </div>
+  </section>
+
+  <!----Projects Section---------->
+
+  <section id="projects">
+    <div class="projects">
+      <h2 class="project_title">
+        My Recent <span class="span__br"><br /></span>Works
+      </h2>
+      <div class="projects_h2_icon"></div>
+      <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
     </div>
+  </section>
+
+  <!-----About Me--------->
+
+  <section id="about">
+    <div class="about_me">
+      <div class="about_container">
+        <h2 class="about__title">About me</h2>
+
+        <p class="about_description">
+          I can help you build a product , feature or website
+           Look through some of my work <span class="span__br2"><br /></span> 
+           and experience! If you like what you see<span class="span__br2"><br /></span>
+            and have a project you need coded, don’t hesitate to contact me.
+        </p>
```

Comment:
- [ ] [Optional] Great job on the about us section! 👍 However, there is a small adjustment that needs to be made to align with the design. The line height should be 32px, but currently, it is set to 28px. Please update the line height to match the design requirement of 32px. This will ensure consistency and visual harmony with the overall design. Keep up the good work! 💪